app/main/views/g_cloud.py

```diff
@@ -265,3 +275,143 @@ def search_services():
         'search/services.html',
         **template_args
     )
+
+
+@direct_award.route('/save-search', methods=['GET'])
+def save_search():
+    # if there are multiple live g-cloud frameworks, we must assume the same filters work on them all
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, 'g-cloud')
+    lots_by_slug = framework_helpers.get_lots_by_slug(framework)
+    current_lot_slug = get_lot_from_request(request, lots_by_slug)
+
+    # the bulk of the possible filter parameters are defined through the content loader. they're all boolean and the
```

Comment:
We shouldn't repeat the comment from the other file. If the code can be shared that'd be best (and the comment moved to the shared place); otherwise just say "see `module.function`"

Reply:
👍